homeassistant/components/smartthings/climate.py

```diff
@@ -68,13 +69,15 @@
     "heat": HVAC_MODE_HEAT,
     "heatClean": HVAC_MODE_HEAT,
     "fanOnly": HVAC_MODE_FAN_ONLY,
+    "wind": HVAC_MODE_WIND,
 }
 STATE_TO_AC_MODE = {
     HVAC_MODE_HEAT_COOL: "auto",
     HVAC_MODE_COOL: "cool",
     HVAC_MODE_DRY: "dry",
     HVAC_MODE_HEAT: "heat",
     HVAC_MODE_FAN_ONLY: "fanOnly",
+    HVAC_MODE_WIND: "wind",
```

Comment:
This is not an existing HVAC mode:

<https://developers.home-assistant.io/docs/core/entity/climate#hvac-modes>